lib/StackAnalysis/StackAnalysis.cpp

```diff
@@ -801,6 +801,38 @@ void commitToModel(GeneratedCodeBasicInfo &GCBI,
   revng_check(TheBinary.verify(true));
 }
 
+static void interproceduralPropagation(const std::vector<CFEP> &Functions,
```

Comment:
I think we can squash this in whatever previous commits makes sense. It's not so crazily important.